docs/source/pages/documentation.md

```diff
@@ -1,18 +1,170 @@
+```
+
+See [the Rclone website](https://rclone.org/install/) for alternative installation methods.
+
+
+### Getting Started
+
+Datashuttle provides a Python API and cross-platform command line interface (CLI). In this guide examples will be down using the command line, but corresponding methods can be found in the [API Reference](https://datashuttle.neuroinformatics.dev/pages/api_index.html).
+
+The first thing to do when using DataShuttle is to setup a new project on a *local* machine.
+
+#### *local* machines and the *central* machine
+
+DataShuttle makes the distinction between (possibly multiple) *local* machines and a single *central* machine. DataShuttle needs to be setup once for each *local* machine, but requires no setup on the *central* machine.
+
+A typical use case is an experment in which behavioural data and electrophysiological data are collected on acquisition PCs. They send the data to a central server where it is stored.
+
+Later, a subset of the data is transferred to a third machine for analysis. In this case, the behavioural and electrophysiological acquisition machine and analysis machines are 'local'. The central storage machine is the *central* machine.
+
+#### One-time project setup on a *local* machine
+
+A one-time setup on each *local* machine used is required, specifying the *project name* and *configurations*.
+
+The configurations tell DataShuttle
+
+- The paths to the *local* and *central* folders that contain the project.
+- How to connect to the central project.
+- The settings that specify how data is transferred.
+- The *data-types* (e.g. *behaviour* (`behav`), *electrophysiology* (`ephys`)) that will be used in the project.
+
+The command `make-config-file` is used for the initial setup of the project. The **required arguments** are:
+
+`local_path`: The full filepath to the project folder on the *local* machine. For example, if you wanted to make a new project called `my_first_project` in the folder `C:\User\my_projects`, the local path would be `C:\User\my_projects`.
+
+`central_path`: The path on the *central* machine to the central project. For example, if connecting to a remote linux server, this may be `/hpc/home/user/my_projects`.
+
+`connection_method`: `local_filesystem` or `ssh`. Local filesystem can be used if the *central* storage is mounted to the local machine. Otherwise `ssh` can be used.
+
+**Optional Arguments**
+
+If connection method is `ssh`, the `central_host_id`, `central_host_username` must be set, and a one-time SSH setup command run (see the [SSH section][#### SSH] for details).
+
+The optional arguments `ovewrite_old_files`, `transfer_verbosity` and `show_transfer_progress` determine how *data transfer* is performed (see the [Data Transfer section](#### Data Transfer) for details).
+
+Finally, the optional *data-type* flags `--use_ephys`, `--use_funcimg`, `--use_histology`, `--use_behav` set the types of data required for the project on the local machine.
```

Comment:
At least one of these must be used though right (from my user testing experiences) - so are these optional?

Reply:
That's correct thanks, I have moved it to the end of the above section, so now it reads:

```
Finally, the *data-type* flags `--use_ephys`, `--use_funcimg`, `--use_histology`, `--use_behav` set the types of data required for the project on the local machine. While individual flags are optional, at least one must be chosen when initialising the project.

**Optional Arguments**
```

Does this seem clear?